lib/turn.rb

```diff
@@ -0,0 +1,121 @@
+class Turn
+  attr_reader :player1, :player2, :spoils_of_war, :loser, :turn_winner
+  def initialize(player1, player2)
+    @player1 = player1
+    @player2 = player2
+    @spoils_of_war = []
+    @turn_winner = nil
+    @loser = loser
+  end
+  
+  # Return boolean value if either player has fewer than 3 cards left
+  
+  def fewer_than_3_cards?
+    if player1.deck.cards.count < 3 || player2.deck.cards.count < 3
+      true 
+    else
+      false
+    end
+  end
```

Comment:
I really like the premise of this method and checking if a player even has enough cards to continue, just want to suggest how you can make this method more succinct. Since we know that `player1.deck.cards.count < 3 || player2.deck.cards.count < 3` will either return `true` or `false`, we can use that to our advantage and refactor this method to look like so:

```ruby
def fewer_than_3_cards?
  player1.deck.cards.count < 3 || player2.deck.cards.count < 3
end
```

Once this method runs, the result will either be `true` or it will be `false`.